### PR TITLE
Update Wi-Fi how-to to include seedimage

### PR DESCRIPTION
**Pull request by juadk**
Just few words to explain that is possible to also configure Wi-Fi in the ISO using cloud-config setup in SeedImage.
It could be needed in certain environment where wired network is not deployed.

### CODE REVIEW

File: docs/wifi.md [hunk 1]
```diff
@@ -6,6 +6,14 @@ title: ''
 
 ### How to configure Wi-Fi
 
+:::caution important note
+We will talk about configuring Wi-Fi inside the new operating system freshly installed in the hard-drive.
+For some reasons, like no wired network connection in the area, you may want to also use Wi-Fi while booting on the ISO.
```
Comment (nunix) [inline docs/wifi.md:11]:
```suggestion
This guide describes how to configure Wi-Fi on a freshly installed operating system.
For some reasons, like no wired network connection available, you may want to also use Wi-Fi when you boot from the ISO.
```

**Review by kkaempf — APPROVED** [review]
:+1:

File: docs/wifi.md [hunk 2]
```diff
@@ -6,6 +6,14 @@ title: ''
...
+
+This is possible by using cloud-config definition in the [SeedImage](https://elemental.docs.rancher.com/seedimage-reference/) resource.
+Report the same instructions as below to create it.
```
Comment (nunix) [inline docs/wifi.md:14]:
```suggestion
This is possible by using a cloud-config definition in the [SeedImage](https://elemental.docs.rancher.com/seedimage-reference/) resource.
You can refer to the instructions below on how to create it.
```

**Review by nunix — COMMENTED** [review]
To be added in a follow-up PR.

**Pull request merged**